components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderConstants.java

```diff
@@ -24,67 +24,138 @@ private SAMLSSOServiceProviderConstants() {
 
     }
 
+    public static class SAML2TableColumns {
+        private SAML2TableColumns() {
+
+        }
+
+        //IDN_SAML2_SSO table
```

Comment:
Add a space after `//`

Reply:
Resolved in [Update SAMLSSOServiceProviderDAOImpl and SAMLSSOServiceProviderConsta…](https://github.com/wso2/carbon-identity-framework/pull/5887/commits/f02e487e0cdf790f28d84ff94eab068d4d397e32)